coeus-impl/pom.xml

```diff
@@ -668,5 +668,21 @@
             <groupId>com.thoughtworks.xstream</groupId>
             <artifactId>xstream</artifactId>
         </dependency>
+
+        <dependency>
```

Comment:
I think these are at the correct scope yes. They are all pulled in jars required for running the app. As far as other dependencies...I dunno. I imagine we can clean this up after if there are any conflicts.